src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,56 @@
 import React from 'react';
-
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  movie: Movie
+};
+
+export const MovieCard: React.FC<Props> = ({ movie }) => {
+  const {
+    imgUrl,
+    title,
+    description,
+    imdbUrl,
+  } = movie;
+
+  return (
+    <div className="card" data-cy="Movie">
+      <div className="card-image">
+        <figure className="image is-4by3">
+          <img
+            data-cy="MovieImage"
+            src={imgUrl}
+            alt="Film logo"
```

Comment:
Would be better if you add more clearly alt for this image
Like a
```suggestion
            alt="`${title} logo`"
```